jupyterhub/values.yaml

```
@@ -47,6 +47,10 @@ hub:
     password:
   labels: {}
   annotations: {}
+  command: jupyterhub
```

I recall there is a recommendation to always use an array rather than a string, I don't remember why but let's go with that unless someone say the opposite.

```suggestion
  command:
    - "jupyterhub"
```